examples/py_proto_library/MODULE.bazel

```diff
@@ -14,11 +14,10 @@ local_path_override(
 
 python = use_extension("@rules_python//python/extensions:python.bzl", "python")
 python.toolchain(
-    name = "python3_9",
     configure_coverage_tool = True,
     python_version = "3.9",
 )
-use_repo(python, "python3_9")
+use_repo(python, "python_3_9")
```

Comment:
So if we register 3.9.11, what is the name? I assume it is 3_9_11.  We need to document this pattern somewhere.   Update the documentation in the tag class and also update the documentation in the MODULE.bazel files.  How the names generated gave me heartburn when I had less bazel experience.

Reply:
We don't support registering patch versions, just major.minor.

I added doc to the tag class.

Which module file are you referring to? I'm guessing the bzlmod example? I've updated the comment there about the name.